src/app/client/src/app/modules/learn/components/course-page/course-page.component.ts

```diff
@@ -436,6 +436,20 @@ export class CoursePageComponent implements OnInit, OnDestroy, AfterViewInit {
     }, {});
     this.dataDrivenFilterEvent.emit(defaultFilters);
   }
+
+  // Generate filters for search API
+  public getSearchFilters(filters) {
+    delete filters['selectedTab'];
+    const filterObj = {
+      'primaryCategory': ['Course', 'Course Assessment'],
```

Comment:
This should not be added, The couse page should work for any trackable primary category. In feature if we have any new trackable content also, it should work.